ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/OracleLinux.yml

```diff
@@ -59,3 +59,23 @@
   shell: cd /tmp/expat-2.2.5 && ./configure && make -j {{ ansible_processor_vcpus }} && sudo make install
   become: yes
   tags: expat
+
+############################
+# Test tools               #
+############################
+- name: Create Japanese locale
+  shell: localedef -i ja_JP -c -f UTF-8 ja_JP.UTF-8
+  tags: test_tools
```

Comment:
What will localedef do if the locale is already defined?

Reply:
It will overwrite with the same data. But, I added to check available locale to avoid the situation.